app/src/main/java/de/westnordost/streetcomplete/osm/Freeform.kt

```diff
@@ -0,0 +1,60 @@
+package de.westnordost.streetcomplete.osm
+
+/** Returns true if value is from some well defined set of values
+ *  Return false otherwise
+ *  Note that it returns false where StreetComplete is using a limited subset of values
+ *  even if tag is effectively freeform for OSM mappers in general
+ *  For example building=* is not considered as freeform.
+ *  Note that freeform tags where SC is using dedicated subset of values but tag is completely freeform
+ *  for example source:maxheight where sole value set by StreetComplete is "ARCore"
+ * */
+fun isTagValueFromWellDefinedSet(key: String, value: String): Boolean {
```

Comment:
Wrong thinking? Shouldn't this function be called isTagValueNOTFromWellDefinedSet?

(But then, better call is `isFreeformTagValue`)

Reply:
I initially named it this way: but for example `building` is effectively freeform in OSM but for SC purposes it is a bit different and individual values can be linked as SC is using specific set of values.

While `opening_hours` is not freeform but linking its values is doomed to fail.

Maybe it should be rather `isTagValueEditedByStreetCompleteLinkableToWiki` or something similarly specific to how it is being used?

Reply:
`isTagValueLinkableToWiki` would be fine. But again, the function currently returns the exact opposite of what the function sounds like then.